extensions/amp-render/1.0/amp-render.js

```diff
@@ -150,10 +177,21 @@ export class AmpRender extends BaseElement {
     });
 
     return dict({
-      'getJson': getJsonFn(this.element),
+      'getJson': this.getFetchJsonFn(),
     });
   }
 
+  /** @override */
+  mutationObserverCallback(mutations) {
+    const isSrcMutated = mutations.some(
+      (mutation) => mutation.attributeName === 'src'
+    );
+    const src = this.element.getAttribute('src');
```

Comment:
Let's store `this.src_` and perform a comparison rather than filtering through these entries.